nuget/dotnet/rl.net.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <title>RL.Net</title>
+    <id>RL.Net</id>
+    <tags>vw vowpal wabbit langford ml machine learning</tags>
+    <authors>John Langford et al</authors>
+    <version>$RL_NUGET_PACKAGE_VERSION$</version>
+    <projectUrl>https://github.com/VowpalWabbit/reinforcement_learning</projectUrl>
+    <license type="expression">MIT</license>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Interaction-side integration library for Reinforcement Learning loops: Predict, Log, [Learn,] Update</description>
+    <copyright>Copyright © Microsoft Corp 2018-2022, and many individual contributors. All rights reserved.</copyright>
+
+    <dependencies>
+      <group targetFramework="netstandard2.0">
+        <dependency id="NETStandard.Library" version="[2.0.0,)" />
+      </group>
+    </dependencies>
+
+  </metadata>
+  <files>
+    <file src="rl.net.dll" target="lib/netstandard2.0" />
```

Comment:
`lib\netstandard2.0` directory is  justfor .NET framework, I believe. It may be the case that .NET Core will magically fall back onto the contents here if it can't find the right directories.

.NET Core should be
* `ref\netstandard2.0` compile-time reference containing managed DLL
* `runtimes\{platform}\lib\netstandard2.0` runtime managed DLL, identical copies for each platform
* `runtimes\{platform}\native` runtime native library that's different for each platform

When I was working on the VW nuget, I also ran into issues with using forward slashes in the .nuspec file. It went away when I changed everything to backslashes. Just a heads up if paths aren't behaving as expected.